pinot-controller/src/main/java/org/apache/pinot/controller/util/ServerSegmentMetadataReader.java

```diff
@@ -49,18 +53,20 @@ public ServerSegmentMetadataReader(Executor executor, HttpConnectionManager conn
    * @return list of segments and their metadata as a JSON string
```

Comment:
(nit) please add javadocs indicating how the column list is used and what is the default behavior

Reply:
done